share/completions/pv.fish

```diff
@@ -0,0 +1,37 @@
+# pv - Pipe Viewer - is a terminal-based tool for monitoring the progress of
+# data through a pipeline.
+# See: http://www.ivarch.com/programs/pv.shtml
+
+complete -c pv -l progress -s p -d 'Turn progress bar on'
+complete -c pv -l timer -s t -d 'Show timer'
+complete -c pv -l eta -s e -d 'Show estimated time left'
+complete -c pv -l fineta -s I -d 'Show estimated time of arrival'
+complete -c pv -l rate -s r -d 'Show rate counter'
+complete -c pv -l average-rate -s a -d 'Show average rate'
+complete -c pv -l bytes -s b -d 'Show total byte counter'
+complete -c pv -l buffer-percent -s T -d 'Show transfer buffer percentage'
+complete -c pv -l last-written -s A -d 'Show last given number of bytes'
```

Comment:
The [man pages](http://www.ivarch.com/programs/quickref/pv.shtml)

>Show the last NUM bytes written

seems okay.